test-status/README.md

```diff
@@ -22,8 +22,7 @@ The App subscribes to the following GitHub Webhooks:
 ### API for Travis
 
 The App has the following API points, which are to be triggered from Travis CI
-runs. If the `TRAVIS_IP_ADDRESSES` environment variable is set, only requests
-from this comma separated list of IP addresses will be processed.
+runs. The IP address of the request is validated against a list of known Travis IP addresses.
```

Comment:
80 chars per line

Reply:
Whoops, assumed eslint would fix that for me, guess not. Fixed.